internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -86,6 +92,18 @@ func NewApplicationSnapshotImage(ctx context.Context, url string, p policy.Polic
 		return nil, err
 	}
 
+	// TODO: At some point, the SignatureValidator is no longer a special case, and it can
+	// be enabled or disabled via the source groups in the PolicyConfiguration.
+	signatureValidator, err := validator.NewImageValidator(
+		signature.SignatureValidatorName, validator.Options{Policy: p})
```

Comment:
Illustrates the point of having named validators, if they did not have a name we could not tie this bit of code to the specific validator, in the end what is the difference than calling a validation method instead? Goes through a map to get to the method?